src/com/lftechnology/java/training/srijan/basics/Fabonacci.java

```diff
@@ -0,0 +1,88 @@
+package com.lftechnology.java.training.srijan.basics;
+
+import java.util.logging.Logger;
+
+/**
+ * Each new term in the Fibonacci sequence is generated by adding the previous
+ * two terms. By starting with 1 and 2, the first 10 terms will be: 1, 2, 3, 5,
+ * 8, 13, 21, 34, 55, 89, ... By considering the terms in the Fibonacci sequence
+ * whose values do not exceed four million, find the sum of the even-valued
+ * terms.
+ */
+
+public class Fabonacci {
+	private int firstNumber;
+	private int secondNumber;
+	private final int numberLimit = 4000000;
+	private int[] fabonacciSeries = new int[4000000];
```

Comment:
At the first look `new int[4000000];` looks fine. But, We are dealing with the Fibonacci series with sum less than `4000000`, So, If we count the Fibonacci it would be around 5000 at max. So, There will be 39950 of unused allocation left in the array. So, That will be ridiculously bad.

This is why we need a dynamic container to hold these values as In Java, "normal" arrays are fixed-size. You have to assign them a size and can't expand them or shrink them. 

So, You need to figure out how to reduce the allocation space for now. Or, You can look into Java List but I guess it is not on the scope right now.